_specs_/websocket/schemas/crypto/actions/aggregates.yml

```
@@ -0,0 +1,26 @@
+SubscribeAction:
+  allOf:
+    - $ref: '../../index.yml#/SubscribeAction'
+    - properties:
+        params:
+          pattern: /^XA\.((?<from>([A-Z]{3})-(?<to>[A-Z]{3}))|\*)$/
```

The `from` group isn't necessarily 3 characters for crypto tickers (ie `DOGE-USD`) 

`to` should always be 3 characters though
```suggestion
          pattern: /^XA\.((?<from>([A-Z]*)-(?<to>[A-Z]{3}))|\*)$/
```